packages/manager/modules/vrack-services/src/routing.js

```diff
@@ -1,47 +1,64 @@
-// import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import { ApiV2ListHelper } from '../../ng-apiv2-helper/src';
 
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('vrack-services', {
     url: '/vrack-services',
-    template: `<div>
-             <div data-ui-view></div>
-        </div>`,
+    template: `<div data-ui-view></div>`,
     redirectTo: 'vrack-services.index',
     resolve: {
       breadcrumb: () => 'vrack-services',
+      apiPath: () => '/vrackServices/resource',
+      createItemsPromise: /* @ngInject */ ($q, apiPath, Apiv2Service) => ({
```

Comment:
Can you rename the createItemsPromise with resourcesPromise or something which makes sense ?

Reply:
> Can you rename the createItemsPromise with resourcesPromise or something which makes sense ?

It's something needed by apiv2 module which come from Iam code so I keep the same label to be homogen